rmi/servercode/ResImpl/MiddleWareImpl.java

```diff
@@ -62,7 +62,12 @@ public static void main(String[] args) {
             System.exit(1);
         }
         // Associate inputted machines to respective RMs
```

Comment:
Our RM's now have a public String banner. That'll either be "Cars", "Flight", or "Hotels"